README.md

```diff
@@ -15,18 +16,91 @@ Let us know how to improve this library. We'll be more than happy if you report
   - [setup & usage](#webhooks---setup--usage)
   - [event types](#webhooks---event-types) 
 - [Missing a feature?](#missing-a-feature)
-  
-## Prerequisities
 
-You need to [create an account](https://app.amio.io/signup) before you can use this library.
+## Prerequisities
+[Signup to Amio](https://app.amio.io/signup) and create a channel before using this library.
 
 ## Installation
 
 ```bash
 npm install amio-sdk-js --save
 ```
 
-## API 
+## Examples
+Send a message:
+
+```js
+const AmioApi = require('amio-sdk-js').AmioApi
+
+// setup AmioApi
+const amioApi = new AmioApi({
+    accessToken: 'get access token at https://app.amio.io/administration/settings/api'
+})
+
+// build a message content using ContentBuilder
+const content = amioApi.contentBuilder.typeText('test message').build()
+
+// send the message
+amioApi.messages.send(
+  {
+    channel: {id: 'get CHANNEL_ID at https://app.amio.io/administration/channels/'},
+    contact: {id: 'get CONTACT_ID at https://app.amio.io/administration/channels/{CHANNEL_ID}/contacts'},
+    content: content,
+  }
+).then(function (message) {
+  console.log(message)
+}).catch(function (error) {
+  console.log(error);
+}).then(function () {
+  // always executed
+}) 
+
+// Want to use async/await? Add the `async` keyword to your outer function/method.
+async function sendMessage() {
+  try {
+    const message = await amioApi.messages.send(
+      {
+          channel: {id: '{CHANNEL_ID}'},
+          contact: {id: '{CONTACT_ID}'},
+          content: content,
+        }
+    )
+    console.log(message);
+  } catch (error) {
+    console.error(error);
+  }
+}
+```
+
+Receive a message:
+```js
+const express = require('express')
+const router = express.Router()
+const WebhookRouter = require('amio-sdk-js').WebhookRouter
+
+const amioWebhookRouter = new WebhookRouter({
+    secrets: {
+      // !!! CHANNEL_ID must be a string. Numbers can be converted to a different value !!!
+      '{CHANNEL_ID}':'{SECRET}'
+    }
+})
+
+// error handling, e.g. x-hub-signature is not correct
+amioWebhookRouter.onError(function(error) {
+  console.error(error)
+})
+
+// assign event handlers 
+amioWebhookRouter.onMessageReceived(function(data) {
+  console.log('new message received from contact ' + data.contact.id + 'with content ' + data.content)
+})
+
+router.post('/webhooks/amio', function (req, res) {
+  amioWebhookRouter.handleEvent(req, res)
+})
+
+module.exports = router
+```
 
 ### API - setup & usage
```

Comment:
Myslím, že stačila úprava jen v téhle a ERrorHandling sekci. :) Každopádně sem ještě pls přidej tohle:

### API - setup & usage
```
const AmioApi = require('amio-sdk-js').AmioApi

const amioApi = new AmioApi({
  accessToken: 'get access token from https://app.amio.io/administration/settings/api'
})

// request with async/await
const message = await amioApi.messages.send({/* message */})

// request with a promise
amioApi.messages.send({/* message */})
 .then(message => ...)
```

### API - Error handling
Amio API errors keep the structure described in the docs.

**With async/await**
```
try{
  await  // ...
} catch(err){
    if (err.amioApiError) {
      console.error(err.jsonify(), err) 
      return
    }
    
    console.error(err) 
}
```

**With promises:**
```
.catch(err => {
if (err.amioApiError) {
    console.error(err.jsonify(), err)
    return
  }

  console.error(err)
})
```